src/components/o-modal/index.js

```diff
@@ -0,0 +1,63 @@
+import PropTypes from '../../js/prop-types';
+import BaseComponentGlobal from '../../js/abstract/base-component-global';
+import defineOnce from '../../js/define-once';
+import styles from './index.scss';
+import template from './_template';
+
+class AXAModal extends BaseComponentGlobal {
+  static tagName = 'axa-modal';
+  static propTypes = {
+    open: PropTypes.bool,
+  };
+
+  init() {
+    super.init({ styles, template });
+  }
+
+  static get observedAttributes() {
+    return ['open'];
+  }
+
+  connectedCallback() {
+    super.connectedCallback();
+
+    const axaButtons = document.querySelectorAll('axa-button[data-modal-close]');
+    if (axaButtons && axaButtons.length) {
+      axaButtons.forEach((button) => {
```

Comment:
some browser wont work with forEach as it does NOT return a native array. It returns a "A non-live NodeList" https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelectorAll#Return_value

You can transform it like so: `export const querySelectorAll = (selector, parent = document) => [].slice.call(parent.querySelectorAll(selector))`